apps/src/openni_3d_concave_hull.cpp

```diff
@@ -131,7 +131,7 @@ class OpenNI3DConcaveHull {
   void
   run()
   {
-    pcl::OpenNIGrabber interface{device_id_};
+    pcl::OpenNIGrabber interface(device_id_);
```

Comment:
Is it me or is device_id_ not setable, but only "" as default value?

Reply:
You are right, [the usage text](https://github.com/PointCloudLibrary/pcl/blob/master/apps/src/openni_3d_concave_hull.cpp#L168) even says that it is possible to specify a device id. The main function should pass the device id (if given by the user) from `argv` to the `OpenNI3DConcaveHull` constructor.
When no device id is given (`""`), `OpenNIGrabber` seems to select the first available device

Reply:
You are right. I still found an old Kinect in storage, unfortunately my system seems to be too new for it. This app crashes with an exception (nice that a PCL exception is generated, which is then not caught), so I could not really test it. However, a functional change is a bit out of scope of this MR, so I would avoid any non-stylistic changes here.

Reply:
I agree, let's create an issue for this.

Out of interest, what was the exception? Where should it be caught, in your opinion?